static/elements/chromedash-color-status.html

```diff
@@ -1,31 +1,43 @@
 <link rel="import" href="../bower_components/polymer/polymer.html">
 
-<polymer-element name="chromedash-color-status" attributes="value max"><!-- width height corner">-->
+<dom-module id="chromedash-color-status">
+  <link rel="import" type="css" href="../css/elements/chromedash-color-status.css">
```

Comment:
Here and elsewhere, I believe that the [`type` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-type) in a `<link>` should be a valid MIME type, so I'd expect this to be `"text/css"`.